src/charm.py

```diff
@@ -112,6 +120,8 @@ def _on_install(self, event: InstallEvent) -> None:
             self._legacy_manager.write_jwt_rsa(jwt_rsa)
 
             self._slurmctld.enable()
+            self._slurmctld.munge.enable()
+            self._slurmctld.exporter.enable()
```

Comment:
[foresight]: we'll want to make the port the prometheus exporter uses configurable in the future as we'll potentially have multiple exporters running per-node. We should likely address this after we have a better idea of what exporters we'll need to have installed on each of the nodes.